Lib9c/Action/ArenaBattle.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume.Battle;
+using Nekoyume.Extensions;
+using Nekoyume.Helper;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
```

Comment:
A summary is needed to list the history of action changes.

```suggestion
{
    /// <summary>
    /// Introduced at: https://github.com/planetarium/lib9c/pull/1006
    /// </summary>
    [Serializable]
```